tests/integration/components/gleam-logo-test.js

```diff
@@ -0,0 +1,134 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'codecrafters-frontend/tests/helpers';
+import { render, settled } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+// Mock Rive class for testing
+class MockRive {
+  constructor(options) {
+    this.canvas = options.canvas;
+    this._listeners = new Map();
+    this._onLoadCallback = null;
+  }
+
+  off(event, callback) {
+    const listeners = this._listeners.get(event);
+
+    if (listeners) {
+      const index = listeners.indexOf(callback);
+
+      if (index > -1) {
+        listeners.splice(index, 1);
+      }
+    }
+  }
+
+  on(event, callback) {
+    if (!this._listeners.has(event)) {
+      this._listeners.set(event, []);
+    }
+
+    this._listeners.get(event)?.push(callback);
+  }
+
+  resizeDrawingSurfaceToCanvas() {
+    // No-op for test
+  }
+
+  simulateLoad() {
+    if (this._onLoadCallback) {
+      this._onLoadCallback();
+    }
+  }
+
+  stateMachineInputs() {
+    return [
+      {
+        name: 'Hover',
+        value: false,
+        type: 'boolean',
+      },
+    ];
+  }
+
+  stop() {
+    // No-op for test
+  }
+
+  triggerEvent(event, ...args) {
+    const listeners = this._listeners.get(event);
+
+    if (listeners) {
+      listeners.forEach((callback) => callback(event, ...args));
+    }
+  }
+}
+
+module('Integration | Component | gleam-logo', function (hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function () {
+    this.originalRive = window.Rive;
+    window.Rive = class MockRiveConstructor {
+      constructor(options) {
+        const instance = new MockRive(options);
+        this._onLoadCallback = options.onLoad;
+
+        return instance;
+      }
+    };
+  });
+
+  hooks.afterEach(function () {
+    window.Rive = this.originalRive;
+  });
+
+  test('it renders and initializes correctly', async function (assert) {
+    await render(hbs`<GleamLogo @height={{200}} />`);
```

Comment:
We're passing in `height` here, but the argument is called `size` in the component file